src/mscorlib/model.xml

```diff
@@ -7534,11 +7534,17 @@
       <Member Name="#ctor" />
       <Member Name="get_IsSingleByte" />
       <Member Name="GetByteCount(System.Char*,System.Int32)" />
+      <Member Name="GetByteCount(System.Char[])" />
       <Member Name="GetByteCount(System.Char[],System.Int32,System.Int32)" />
       <Member Name="GetByteCount(System.String)" />
+      <Member Name="GetByteCount(System.String,System.Int32,System.Int32)" />
       <Member Name="GetBytes(System.Char*,System.Int32,System.Byte*,System.Int32)" />
       <Member Name="GetBytes(System.Char[],System.Int32,System.Int32,System.Byte[],System.Int32)" />
       <Member Name="GetBytes(System.String,System.Int32,System.Int32,System.Byte[],System.Int32)" />
+      <Member Name="GetBytes(System.Char[])" />
+      <Member Name="GetBytes(System.Char[],System.Int32,System.Int32)" />
+      <Member Name="GetBytes(System.String)" />
+      <Member Name="GetBytes(System.String,System.Int32,System.Int32)" />
```

Comment:
why we didn't add the new introduced APIs to Encoding class? Also looks not all of these methods implemented in ASCIIEncoding